test/uk/gov/hmrc/economiccrimelevyregistration/connectors/EclRegistrationConnectorSpec.scala

```diff
@@ -144,4 +146,90 @@ class EclRegistrationConnectorSpec extends SpecBase {
         reset(mockHttpClient)
     }
   }
+
```

Comment:
Are we missing a test when the subscription id does not exist (HTTP 404)?

Reply:
No - since we rely on the library UpstreamErrorResponse handling for that, like our other connectors